04/components/Nav.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Menu from "./Menu";
+const props = {
+  home: {
+    id: 1,
+    text: "Home",
+    url: "/",
+  },
+  products: {
+    id: 2,
+    text: "Products",
+    url: "/products",
+  },
+  contact: {
+    id: 3,
+    text: "Contact",
+    url: "/contact",
+  },

```

Comment:
Pytanie czy nie wygodniej (i elastyczniej) zrobić z tego po prostu tablicę - nie interesuje nas jakie elementy to są, menu to menu - wystarczą pozycję czyli tablica elementów (o odpowiedniej strukturze tj. obiekcie).

Obecnie jeśli dodasz kolejną pozycję to musisz to wziąć pod uwagę w `Menu`, przy samej tablicy nie musiałbyś tego robić.